google_guest_agent/agentcrypto/mtls_mds_windows.go

```diff
@@ -59,6 +59,11 @@ var (
 
 // writeRootCACert writes Root CA cert from UEFI variable to output file.
 func (j *CredsJob) writeRootCACert(_ context.Context, cacert []byte, outputFile string) error {
+	num, err := serialNumber(outputFile)
```

Comment:
Can we move this closer to line 92? It only makes sense there, the logging can also be done within that if statement.

Reply:
if we receive new creds, this file will be overwritten by that time. So we need to make sure previous one is read before we attempt to write

Reply:
Can you add a comment explaining that?

Reply:
Done